metadata-ingestion/src/datahub/ingestion/source/looker/looker_template_language.py

```diff
@@ -92,52 +108,211 @@ def resolve_liquid_variable(text: str, liquid_variable: Dict[Any, Any]) -> str:
     return text
 
 
-def _drop_derived_view_pattern(value: str) -> str:
```

Comment:
add some comments at the top that explain what's going on, similar to the PR description